### PR TITLE
TW-76 Scratch Pad docs

### PR DESCRIPTION
**Pull request by jkonrath-postman**
* New page
* Modifications to the syncing and managing account pages

### CODE REVIEW

File: src/pages/docs/getting-started/using-scratch-pad.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+title: "Using the Scratch Pad"
+order: 7.1
+page_id: "using-the-scratch-pad"
+warning: false
+---
+
+The _Scratch Pad_ is a space where you can work while not being connected to Postman Servers. When you aren't logged in or don't have a network connection, you can still access some Postman features offline, such as creating collections and requests, or sending requests.
+
+All work in the Scratch Pad will be stored locally and will not be synced online with Postman. After working in the Scratch Pad, you can later move your work into a workspace after you log in.
+
```
Comment (sowmyaraj92) [inline src/pages/docs/getting-started/using-scratch-pad.md:11]: All work in the Scratch Pad > "All the/your work in the Scratch Pad"
after you log in > "once you log in"

**Review by sowmyaraj92 — CHANGES_REQUESTED** [review]
@jkonrath-postman Looks great overall, just minor edits! 🎉

File: src/pages/docs/getting-started/using-scratch-pad.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+If you're logged out or your connection to Postman is broken, you'll see a notice at the top of the screen indicating that you're already in the Scratch Pad.
+
+<img alt="Scratch Pad notice" src="https://assets.postman.com/postman-docs/scratch-pad-notice.jpg" width="350px" />
+
+If you are logged in and want to go to the Scratch Pad, click the gear icon and select **Scratch Pad**.
+
+<img alt="Enter Scratch Pad" src="https://assets.postman.com/postman-docs/scratch-pad-enter.jpg" width="200px" />
+
+## What you can do in the Scratch Pad
```
[the remark this answers is not in the document]
Comment (jkonrath-postman) [reply src/pages/docs/getting-started/using-scratch-pad.md:22]: Good idea, I'll use that.

File: src/pages/docs/getting-started/using-scratch-pad.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+While you are in the Scratch Pad, you can:
+
+* Create, edit, rename, or delete a collection, folder, environment, HTTP request, or WebSocket request.
+* Send a request.
+* View your history of sent requests.
+* Write and execute test scripts and visualizers.
+* Export collections and environments.
+* Edit documentation for a collection or request.
```
[the remark this answers is not in the document]
Comment (jkonrath-postman) [reply src/pages/docs/getting-started/using-scratch-pad.md:31]: If a list is a mix of complete and incomplete sentences, you use a period for all of them. https://docs.microsoft.com/en-us/style-guide/scannable-content/lists#punctuation

File: src/pages/docs/getting-started/using-scratch-pad.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+warning: false
+---
+
+The _Scratch Pad_ is a space where you can work while not being connected to Postman Servers. When you aren't logged in or don't have a network connection, you can still access some Postman features offline, such as creating collections and requests, or sending requests.
+
+All work in the Scratch Pad will be stored locally and will not be synced online with Postman. After working in the Scratch Pad, you can later move your work into a workspace after you log in.
+
+## Entering the Scratch Pad
+
+If you're logged out or your connection to Postman is broken, you'll see a notice at the top of the screen indicating that you're already in the Scratch Pad.
```
[the remark this answers is not in the document]
Comment (jkonrath-postman) [reply src/pages/docs/getting-started/using-scratch-pad.md:14]: Confirming this, but if there's no consensus from UX, I might stick with notice.

[the remark this answers is not in the document]
Comment (jkonrath-postman) [reply src/pages/docs/getting-started/using-scratch-pad.md]: It's "global banner"...

**Review by sowmyaraj92 — APPROVED** [review]
Looks great! 😊 Ready to merge 🚀 🎉

File: src/pages/docs/getting-started/using-scratch-pad.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (sowmyaraj92) [reply src/pages/docs/getting-started/using-scratch-pad.md]: Thank you for this information. Much needed :)

**Pull request merged**